packages/react-native-client/ios/RNFishjamClientModule.swift

```diff
@@ -86,6 +82,30 @@ public class RNFishjamClientModule: Module {
             return client
         }()
 
+        Name("RNFishjamClient")
+
+        Events(EmitableEvents.allEvents)
+
+        Property("peerStatus") {
+            rnFishjamClient.peerStatus.rawValue
```

Comment:
**nitpick:** 
This one also sir.

Reply:
If you are determined you can do it for also RNFishjamClientModule.kt to be consistent.

Reply:
Okay!